WeakAuras/WeakAuras.lua

```diff
@@ -394,10 +394,10 @@ function WeakAuras.ActivateAuraEnvironment(id, cloneId, state)
         local func = WeakAuras.customActionsFunctions[id]["init"];
         if func then
           current_aura_env.id = id;
+          data.init_started = 1;
```

Comment:
Yep. Fixed in [7c9f7c9](https://github.com/WeakAuras/WeakAuras2/pull/217/commits/7c9f7c970235a8d782d46fbea94fcb3ee33f2dab).